test/findings/test_finding_container.py

```diff
@@ -163,6 +169,18 @@ def test_human_readable_message_for_all_findings(self):
             message, "test.proto L1: A new method `subject` is added to service ``.\n"
         )
 
+    def test_suggested_commits_for_all_findings(self):
```

Comment:
Can we add a test that asserts the `len(FindingCategory) == len(suggested_commit_templates)` with a nice message saying "if you add a new FindingCategory value, you must add a suggested_commit_template as well!" ?

This will help us keep them in sync.